### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Meonthb** [tefkah/zotero-night#148]
**Describe the bug**
Night mode don't apply on PDF in Zotero 7

**To Reproduce**
Toggled the night mode icon, only the UI turned into dark, but not the PDF.

**Screenshots**
![image](https://github.com/tefkah/zotero-night/assets/60881723/6bc2a823-291b-42cc-a201-efb992a26401)

**OS (MacOS XX.xx [specify Intel or Silicon], Windows, Linux [specify distro/package type]:**
Windows 11
version: 22H2
OS build: 22621.2215
Experience: Windows Feature Experience Pack 1000.22662.1000.0

**Comment by tefkah** [comment]
Are you using the latest version of the plugin? The first release for Zotero 7 had that problem

**Comment by tefkah** [comment]
thanks for double checking! this might be a windows version problem then, which is kind of hard for me to check. If other people have this problem on Windows 11 please let me know!

**Comment by tefkah** [comment]
Oh and which version of Zotero are you running?

**Comment by tefkah** [comment]
@FlynnSpace Seems like you are using the Zotero 6 version, what version of the plugin of the plugin are you using?

**Comment by tefkah** [comment]
For the people on Zotero 7, please try the latest release

**Comment by FlynnSpace** [comment]
@tefkah The plugin is 0.4.23 and the Zotero is 6.0.27. Should I download the old version plugin or Zotero?

**Comment by jiagengliu** [comment]
I had the same problem on mac, but restarting Zotero solved the problem.

**Comment by zgz292dev** [comment]
I have the same problem on Windows 11. OS build : 22621.228

**Comment by kzssc** [comment]
Mac 13.5.2 
Night 0.1.0-2 2023-09-12 11:55:51
7.0.0-beta.40+24ae34104

Can confirm that sometimes dark mode works for *some* PDF, but not all. 
This can be rectified *not* by restarting zotero but rather closing the document and then re-open from within Zotero.

**Comment by csy94cfy** [comment]
> @tefkah The plugin is 0.4.23 and the Zotero is 6.0.27. Should I download the old version plugin or Zotero?

hey dude, I have the same versions as you, have you solved your question?

**Comment by FlynnSpace** [comment]
> > @tefkah The plugin is 0.4.23 and the Zotero is 6.0.27. Should I download the old version plugin or Zotero?
> 
> hey dude, I have the same versions as you, have you solved your question?

No, the background of all my PDFs still stings the eyes, even if I restart the plugin or Zetero. (╯ ͡❛ ͜ʖ ͡❛)╯┻━┻

**Comment by franzbischoff** [comment]
> > > @tefkah The plugin is 0.4.23 and the Zotero is 6.0.27. Should I download the old version plugin or Zotero?
> > 
> > 
> > hey dude, I have the same versions as you, have you solved your question?
> 
> No, the background of all my PDFs still stings the eyes, even if I restart the plugin or Zetero. (╯ ͡❛ ͜ʖ ͡❛)╯┻━┻

Try the "disable/enable" trick while not fixed :)

**Comment by FlynnSpace** [comment]
@csy94cfy hey bro, I think I found the alternative solution from issue #128 . You can try the plugin "zotero-style", which is able to set the PDF's background to dark.

**Comment by ruturajreddy** [comment]
For me it worked by going to Tools -> Night preferences and enabling disabling the option for dark mode and selecting the option for the default pdf view. Working on Zotero 6.0.37.

This is what it should look like.
<img width="258" alt="Screenshot 2024-04-02 at 5 30 12 PM" src="https://github.com/tefkah/zotero-night/assets/69036453/6598979d-b1d3-4034-85de-61215aae0f6d">
